content/en/standard-attributes/_index.md

```diff
@@ -0,0 +1,1807 @@
+---
+title: Default Standard Attributes
+disable_sidebar: true
+further_reading:
+    - link: '/data_security/'
+      tag: 'Documentation'
+      text: 'Ensuring the security of the data sent to Datadog'
+filter_all: All
+content: "The following table lists the attributes automatically applied to data sent to Datadog by the Agent by each of the RUM, Logs, and APM products, as applicable to the data domain. Optionally, filter the list by product or search by keyword or description text to find the attributes you're interested in."
+
+attributes:
```

Comment:
Is there a way to bump up the padding on the page so the metric description table isn't cut off slightly?

![image](https://github.com/DataDog/documentation/assets/76412946/7cf96d70-bf42-445f-b2da-227a757246bc)

Reply:
you should be able to stroll the table to the left or right, but I'll tinker with the table a bit